pkg/ebpf/tracee.go

```diff
@@ -122,6 +123,8 @@ type Tracee struct {
 	streamsManager *streams.StreamsManager
 	// policyManager manages policy state
 	policyManager *policyManager
+	// producer produce events in analyze mode instead of eBPF programs
+	producer producer.EventsProducer
```

Comment:
Perhaps not relevant for this PR, but I see a future where this is a `one-tracee-many-producers` relation. @josedonizetti WDYT?